railties/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+*   Create a new rails app using a minimal stack.
+
+      `rails new cool_app --minimal`
+
+    All the following are removed from your minimal stack:
```

Comment:
May I suggest using "not added" instead of "removed"? The setup is disabling options (or removing default components) resulting in blocks not being added to the stack. But it's not building the app then removing pieces of that stack.

It's a nitpick so feel free to ignore, but because this is the main description in the changeling, it might end up propagating to the `help` menu and the documentation, so it could be important to keep the copy accurate.

Reply:
Thanks for the review, I have updated the change log text to `excluded`.